app/javascript/components/LinkButton.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <button
+    @click="moveMe()"
+    :class="[classmod,'text-white font-bold py-2 px-4 rounded focus:outline-none focus:shadow-outline']"
+    :disabled="fieldDisabled"
+  >{{ fieldPlaceholder }}</button>
```

Comment:
Debiesemos pasar el texto del botón como un children, asi como se hace en react con this.props.children. Buscando en internet vi que se hace asi:
```suggestion
  <button
    @click="moveMe()"
    :class="[classmod,'text-white font-bold py-2 px-4 rounded focus:outline-none focus:shadow-outline']"
    :disabled="fieldDisabled"
  ><slot /></button>
```